linux/package.json

```diff
@@ -13,20 +13,32 @@
     "electron": "./node_modules/.bin/electron .",
     "player": "./bin/player-window.sh",
     "production": "NODE_ENV=production electron .",
-    "build": "NODE_ENV=production NEW_VERSION=$npm_package_version ./bin/build.sh",
-    "postinstall": "electron-rebuild"
+    "package": "NODE_ENV=production NEW_VERSION=$npm_package_version ./bin/build_package.sh",
+    "dist": "NODE_ENV=production NEW_VERSION=$npm_package_version ./bin/build_installers.sh",
+    "postinstall": "electron-rebuild",
+    "lint": "eslint .",
+    "mocha": "mocha",
+    "test": "npm run lint && npm run package && mocha test/app.js && npm run dist && mocha test/installers.js"
   },
   "dependencies": {
     "dbus": "^0.2.23",
     "electron-window-state": "^4.1.1",
     "mpris-service": "^1.1.3"
   },
   "devDependencies": {
+    "babel-eslint": "^7.2.3",
     "electron": "^1.7.5",
-    "electron-installer-debian": "^0.5.2",
+    "electron-installer-debian": "fcastilloec/electron-installer-debian#lintian",
```

Comment:
This is a controversial change. I have submitted two PRs to `electron-installer-debian` one has been updated and the second is in the process. Both of the PRs are needed in order for lintian checks to pass when testing the `deb` installer. 
If we use the `npm` version, we can't check for lintian errors/warnings anymore.
Also, the PR that was approved has support for scripts which would make installing a repository automatically